### ISSUE TITLE
Image analysis request results in 500 error

### ISSUE THREAD

**Issue by mayaCostantini** [thoth-station/user-api#1672]
**Describe the bug**

Querying for an existing image analysis with id `analysis_id` on the `analyze/{analysis_id}` endpoint results in a `500` error specifying that the schema could not be validated for the `skopeo-inspect` property. 

Error log:

```
{
  "detail": "'architecture' is a required property\n
	\nFailed validating 'required' in schema['properties']['result']['properties']['skopeo-inspect']:\n
	    {'description': 'Response information with image metadata as extracted '\n
                    'by Skopeo',\n
     'properties': {'architecture': {'description': 'Target architecture '\n
                                                    'of image',\n
                                     'example': 'amd64',\n
                                     'type': 'string'},\n
                    'created': {'description': 'Image creation date and '\n
                                               'time',\n
                                'example': '2016-03-04T18:40:02.92155334Z',\n
                                'type': 'string'},\n
                    'digest': {'description': 'Digest of the image',\n
                               'example': \"sha256:cfd8f071bf8da7a466748f522406f7ae5908d002af1b1a1c0dcf893e183e5b32''\",\n
                               'type': 'string'},\n
                    'docker_version': {'description': 'Version of Docker',\n
                                       'example': '1.9.1',\n                                       'type': 'string'},\n

                    'labels': {'description': 'Image labels',\n                               'type': 'object'},\n
                    'layers': {'items': {'description': 'Digests of image '\n
                                                        'layers',\n
                                         'type': 'string'},\n                               'type': 'array'},\n                    'name': {'description': 'Name of the image with '\n
                                            'optional tag',\n
                             'example': 'fedora',\n
                             'type': 'string'},\n                    'os': {'description': 'Operating system identifier',\n

                           'type': 'string'},\n
                    'repo_tags': {'items': {'description': 'Tags available '\n                                                           'in the '\n                                                           'registry '\n                                                           'repository',\n                                            'type': 'string'},\n                                  'type': 'array'},\n                    'tag': {'description': 'Tag of the image',\n                            'example': 'latest',\n                            'type': 'string'}},\n     'required': ['architecture',\n                  'created',\n                  'digest',\n                  'docker_version',\n                  'labels',\n                  'layers',\n                  'name',\n                  'os',\n                  'repo_tags'],\n     'type': 'object'}\n\nOn instance['result']['skopeo-inspect']:\n    {'Architecture': 'amd64',\n     'Created': '2021-12-14T16:34:17.452175Z',\n     'Digest': 'sha256:be112e49c8addf19842a58ee30203f64d33affca6eb4fb74d6beac19d2f45259',\n     'DockerVersion': '1.13.1',\n     'Env': ['PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',\n             'container=oci'],\n     'Labels': {'architecture': 'x86_64',\n                'build-date': '2021-12-14T16:33:54.544244',\n                'com.redhat.build-host': 'cpt-1008.osbs.prod.upshift.rdu2.redhat.com',\n                'com.redhat.component': 'ubi8-minimal-container',\n                'com.redhat.license_terms': 'https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI',\n                'description': 'The Universal Base Image Minimal is a '\n                               'stripped down image that uses microdnf as '\n                               'a package manager. This base image is '\n                               'freely redistributable, but Red Hat only '\n                               'supports Red Hat technologies through '\n                               'subscriptions for Red Hat products. This '\n                               'image is maintained by Red Hat and updated '\n                               'regularly.',\n                'distribution-scope': 'public',\n                'io.k8s.description': 'The Universal Base Image Minimal is '\n                                      'a stripped down image that uses '\n                                      'microdnf as a package manager. This '\n                                      'base image is freely '\n                                      'redistributable, but Red Hat only '\n                                      'supports Red Hat technologies '\n                                      'through subscriptions for Red Hat '\n                                      'products. This image is maintained '\n                                      'by Red Hat and updated regularly.',\n                'io.k8s.display-name': 'Red Hat Universal Base Image 8 '\n                                       'Minimal',\n                'io.openshift.expose-services': '',\n                'io.openshift.tags': 'minimal rhel8',\n                'maintainer': 'Red Hat, Inc.',\n                'name': 'ubi8-minimal',\n                'release': '218',\n                'summary': 'Provides the latest release of the minimal Red '\n                           'Hat Universal Base Image 8.',\n                'url': 'https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8-minimal/images/8.5-218',\n                'vcs-ref': '7ffc2d965ea414fc38f42d93090d967a5aee2c33',\n                'vcs-type': 'git',\n                'vendor': 'Red Hat, Inc.',\n                'version': '8.5'},\n     'Layers': ['sha256:131f1a26eef08c7274f7a115f461689819a4218e246ccacacc50ebcfd5e1c758',\n                'sha256:16b78ed2e82205ceca54aa3a4205f029096a354d346b3bd2434a56b7f66e21c1'],\n     'Os': 'linux',\n     'RepoTags': []}",
  "status": 500,
  "title": "Response body does not conform to specification",
  "type": "about:blank"
}
```

Similar issue: https://github.com/thoth-station/user-api/issues/1660

**To Reproduce**

Go to the API on stage and query the analysis result for the `analysis_id` `package-extract-220208200628-b8616cf02e7ce891` and see the error.

**Expected behavior**

An image analysis is returned for the given `analysis_id`.

**Comment by mayaCostantini** [comment]
Waiting for `user-api` to be redeployed on stage, but https://github.com/thoth-station/user-api/pull/1676 should solve this issue (and https://github.com/thoth-station/integration-tests/issues/251)

**Comment by mayaCostantini** [comment]
/close
Fixed by #1676
